src/pkg/packager/deploy.go

```diff
@@ -467,9 +470,9 @@ func (p *Packager) processComponentFiles(component v1alpha1.ZarfComponent, pkgLo
 
 		// Copy the file to the destination
 		spinner.Updatef("Saving %s", file.Target)
-		err := helpers.CreatePathAndCopy(fileLocation, file.Target)
-		if err != nil {
-			return fmt.Errorf("unable to copy file %s to %s: %w", fileLocation, file.Target, err)
+		err2 := helpers.CreatePathAndCopy(fileLocation, file.Target)
```

Comment:
why call this err2?

Reply:
A general aversion to var shadowing 👀 i don't mind sticking to err though.

Reply:
I think this syntax should accomplish that, though it's not possible for functions with multiple returns
```go
if err := helpers.CreatePathAndCopy(fileLocation, file.Target); err != nil {
	return fmt.Errorf("unable to copy file %s to %s: %w", fileLocation, file.Target, err)
}
```
IMO best to just use err when we are going to return right away, then use different names when we want to use old errors. That way it's clear to the reader when we're keeping around old errors / avoiding shadowing on purpose.

Reply:
yeah i vibe w/ that